ludwig/modules/metric_modules.py

```diff
@@ -465,7 +465,7 @@ def __init__(self, **kwargs):
         super().__init__()
 
     def update(self, preds: Tensor, target: Tensor) -> None:
-        super().update(preds.detach(), target)
+        super().update(preds, target)
```

Comment:
What's the error if the `preds` tensor is detached?

Reply:
I tried rerunning the scripts that I was testing with and can't reproduce the error. I'll remove this change for now.